controller/src/main/java/io/pravega/controller/metrics/TransactionMetrics.java

```diff
@@ -54,6 +55,7 @@ public final class TransactionMetrics extends AbstractControllerMetrics {
     private final OpStatsLogger abortTransactionLatency;
     private final OpStatsLogger abortTransactionSegmentsLatency;
     private final OpStatsLogger abortingTransactionLatency;
+    private final OpStatsLogger controllerEventProcessorCommitTransactionStreamLatency;
```

Comment:
I will prefer to have name controllerEventProcessorCommitTransactionLatency instead of controllerEventProcessorCommitTransactionStreamLatency.

Reply:
fixed